client/src/store/dogs.js

```diff
@@ -23,7 +23,7 @@ export const getAllDogs = () => {
   return (dispatch) => {
     return axios
       .get(DOGS_API)
-      .then((response) => sortBy(response.data, [(dog) => !dog.available]))
+      .then((response) => sortBy(response.data))
```

Comment:
The old line was there to sort dogs by availability (unavailable dogs appear last). If we don't need to sort anymore, we can remove this line (remember to test before pushing to make sure data is still sane - best if automated tests are written).